php/src/vaas/Options/ForSha256Options.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace VaasSdk\Options;
+
+class ForSha256Options
+{
+    public bool $useCache;
+    public bool $useHashLookup;
+    public ?string $vaasRequestId;
+
+    public function __construct(array $options = [])
```

Comment:
Why do you use an arbitrary array here instead of just parameters for the constructor?